portfolio/src/main/webapp/comments.js

```diff
@@ -17,7 +17,9 @@ function checkCommentPosted() {
   const url = new URL(window.location.href);
   const posted = url.searchParams.get("comment-posted");
   if(posted === "false") {
-    window.alert("Comment not posted!");
+    const reason = url.searchParams.get("reason");
+    const message = (reason === "score") ? "Your sentiment score was too low!" : "Your message was empty!";  
```

Comment:
You can use a switch statement here, since you have separate reasons for both the conditions, to be more clear and to avoid breaking this code in case you add more reasons later.